### PR TITLE
add rules about job postings

### PR DESCRIPTION
**Pull request by ajm188**
approximately 90% confident in the quality of my rst.

### CODE REVIEW

File: job_postings.rst [hunk 1]
```diff
@@ -0,0 +1,24 @@
...
+If you'd like assistance in writing a job post aimed towards students or if you
+have other questions such as expected pay or hours, contact acmpr@case.edu.
+We'd be happy to assist you so that your post gets the maximum response.
+
+Inappropriate job postings will be deleted. While the final decision is at the
+full discretion of the group admin, students are encouraged to report job
+postings they find inappropriate.
+
+Please be aware that many students may have strong opinions on NDAs.
+http://blog.jpl-consulting.com/…/…/why-i-wont-sign-your-nda/
```
Comment (thomasrussellmurphy) [inline job_postings.rst]: Complete link?

File: job_postings.rst [hunk 2]
```diff
@@ -0,0 +1,24 @@
...
+few "house rules" regarding postings:
+
+Be sure to mention the basics such as location, type of development or
+research, and pay. Students are most interested in knowing who they'll be
+working with, what they'll be doing, and what the company does. We ask that you
+save business & funding details for private conversations if the topic
+arises.
+
+If you'd like assistance in writing a job post aimed towards students or if you
+have other questions such as expected pay or hours, contact acmpr@case.edu.
```
Comment (timtadh) [inline job_postings.rst]: maybe just change that to contact us?

Comment (thomasrussellmurphy) [inline job_postings.rst]: Where does `acmpr` go?

Comment (ajm188) [inline job_postings.rst]: no idea. will change

File: job_postings.rst (hunk 1 above)
Comment (ajm188) [inline job_postings.rst]: damn you facebook

**Comment by brenns10** [comment]
rst looks fine. Don't feel like fetching to test on Sphinx ¯\_(ツ)_/¯.  Only question is whether this is really "wiki" material?  I mean, it looks like good advice/rules, but I doubt many job posters care enough about HacSoc to have gotten all the way to our (rather hidden) wiki and get here.  I guess it's good to have to link to as a "here's what you did wrong", but then in that case we may want to expand/formalize do's and don'ts.

**Comment by thomasrussellmurphy** [comment]
Should Facebook have a link to this once it's merged to avoid synchronization issues in our policy?

**Comment by ajm188** [comment]
@thomasrussellmurphy: yes, and I'm a group owner, so I can do that.
@brenns10: if someone reaches out to us with a job posting (it's happening on the officers thread right now), that violates any of these guidelines, we can just point them here rather than spell it out every single time.

**Comment by timtadh** [comment]
@brenns10 I asked for it. It is real wiki material we need this in an accessible location because we get lots of job postings. I want to just be able to drop them a link. Also I am planning on putting a link to the wiki from the main site when I redo it a bit this spring.

**Comment by brenns10** [comment]
Arguments are convincing enough for me.  It's a :shipit: from me unless there's anything left to add.

@timtadh :+1: on putting a wiki link on the main page.  Would be nice.  I was actually thinking of doing that, but my free time is slowly becoming negative now.

**Pull request merged**